wordpress-mobile/WordPress-Android
# Update values associated with "Low" and "Medium" image quality settings

> **Note**
> Although this PR is large, the vast majority of file changes are the result of a change to a string name, as can be viewed in https://github.com/wordpress-mobile/WordPress-Android/pull/19644/commits/452ed672ac3d147571376dc954eb907bb4f34c23.

Merging into https://github.com/wordpress-mobile/WordPress-Android/pull/19581.

## Description

This PR changes the default values associated with the `Low` and `Medium` image quality settings found under _Me_ → _App Settings_.

The `Low` value has been changed from `80` to `70`, and the `Medium` value has been changed from `85` to `80`. This brings the Android app with the values introduced for iOS in https://github.com/wordpress-mobile/WordPress-iOS/pull/21981.

We've decided to change these values without explicitly notifying users, with the following in mind:

- Users don’t technically select `85`. They choose `Medium` or `Low`. The decision around what these values meant was set around 6 or 7 years ago. We think it’s fair to reassess what `Medium` or `Low` means in this day and age (now that image quality tends to be a lot higher, and therefore 80% compression may not be as visually impactful as it would have been years ago).
- 5% isn’t that meaningful a difference, and we wouldn't expect most users who've chosen `Medium` to notice it.
- Those users who've selected `Low` know the route to follow to change this setting, and have already manually indicated that they value speed over quality.

Related discussion: pcdRpT-4tj-p2 

-----

## To Test

- Add an image to your phone's library with a large file size.
- Navigate to _Me_ → _App Settings_ and ensure **Optimize Images** is toggled on.
- Select either **Low** or **Medium** from the **Image Quality** dropdown.
- Navigate to the post editor and add an image block.
- Tap on the **Choose from device** option.
- Select the image previously added with a large file size, and wait for the image upload to finish.
- Go to the browser and navigate to the **Media** screen of the site.
- Select the uploaded image and click on the **Edit** button.
- Observe that the file size under the FILE SIZE section is smaller than the original file size and is of the quality you'd expect.

-----

## Regression Notes

1. Potential unintended areas of impact

    - As we're changing numerous language files, there's a chance translations could be unintentionally impacted.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - I manually tested the app in a few different languages to verify there were no regressions.

3. What automated tests I added (or what prevented me from doing so)

    - Automated tests for the image optimization settings are to follow in a separate PR.

-----

## PR Submission Checklist:
